scripts/tiqit/__init__.py

```diff
@@ -847,7 +835,7 @@ def _loadNamedPage(ofType):
         if not arg.find('=') >= 0:
             break
         key, val = arg.split('=', 1)
-        key, val = map(urllib.unquote, (key, val))
+        key, val = list(map(urllib.parse.unquote, (key, val)))
```

Comment:
Won't let me provide the suggestion: But can do on a single line and avoid using `map` as above